script/QA/QA_20.txt

```diff
@@ -121,72 +127,81 @@ Correct. Depending on the player's perspective,
 [sha:585290c28b9ed6cd1b620209c1ec8d9e20fe0aa4]{
 -- Page 179, Offset 43647.
 -- 幸福なのかに別れる、メリーバッドエンドです。
-this could either be a sad or happy ending. A merry bad end.
+ this could%{n}
+be a sad ending or a happy ending. A merry bad end.
 }
 [sha:190b99337a20e75857c971bcc74ef75d24d6cf3b]{
 -- Page 180, Offset 43648.
 -- にゃる。それで責任感じて
-I see. So Ciel must be feeling down and depressed
+I see. So Ciel must be feeling down in the%{n}
 }
 [sha:de4403fb7daf2373c1883fa1cb30d08f63a60cd3]{
 -- Page 180, Offset 43649.
 -- 落ち込みダウン中なのか、シエル。
-from the weight of her responsibility.
+dumps from the weight of her responsibility.
 }
 [sha:9306f97cb72f8972ad5b04a4907e4cc072c85f5f]{
 -- Page 181, Offset 43650.
 -- でもアタシ、このエンド好きだなー。
-But as for me, I quite like this kind of ending.
+But as for me, I kinda like this ending.%{n}
 }
 [sha:3845facb85d1ac4b218a4c765cdfb7d18994890b]{
 -- Page 181, Offset 43651.
 -- 「これで良かったんだよ」感ある。
-It has that feeling of 'I'm glad it ended like this'.
+It sorta feels like it was for the best.
 }
 [sha:734744e0bd29542de9d3c9097faf76205af0597e]{
 -- Page 182, Offset 43652.
 -- ……そうですね。大局的な視点で見ると
-...You have a point. From a big picture perspective,
+You have a point...
+ From a big picture%{n}
+perspective,
 }
 [sha:a531c8e1a44b58b264e7ca82bd4f761def30725d]{
 -- Page 182, Offset 43653.
 -- この後に出る被害者はゼロです。
-there would be no more other victims after this.
+ there won't be any more victims%{n}
+now.
 }
 [sha:eb98624cd71759f47c48a9b05f507615d49bff1d]{
 -- Page 183, Offset 43654.
 -- 正しい終わり……正解のひとつ、と
-A virtuous ending... it would be wrong to say
+A virtuous ending...
 }
 [sha:40d51f20b0e33df863e9715ff6cd575f7b7e32c4]{
 -- Page 183, Offset 43655.
 -- 言えなくもありません。
-that there was a correct answer here.
+ One could even say%{n}
+that this was one of the correct answers.
 }
 [sha:3b7ca79030313241a02c1cc0cf8c0e234637e4cc]{
 -- Page 184, Offset 43656.
 -- マーリオゥが聞いたら、鼻で笑われそうですが。
-If Mario were to hear about this, he would be snickering right now.
+If Mario could hear us, he would be snickering right%{n}
+about now.
```

Comment:
I think the implication is that Mario doesn't think this is "the correct answer" in the slightest, so something to that effect.
>Though if Mario heard you say that, he'd mock you for it.